ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/EngineExchangeTransitionConfigurationParameter.java

```diff
@@ -46,4 +46,13 @@ public Hash getTerminalBlockHash() {
   public long getTerminalBlockNumber() {
     return terminalBlockNumber;
   }
+
+  @Override
+  public String toString() {
+    return "EngineExchangeTransitionConfigurationParameter{" +
+            "terminalTotalDifficulty=" + terminalTotalDifficulty.toShortHexString() +
+            ", terminalBlockHash=" + terminalBlockHash.toShortHexString() +
+            ", terminalBlockNumber=" + terminalBlockNumber +
+            '}';
+  }
```

Comment:
Switching to Jackson this is not needed, btw `toString` is free form and could not return a valid json if modified in the future, better use something else.

Reply:
I've updated the PR to use Jackson